app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def most_recent_posts
+    posts.oreder(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] Kindly fix this typo error as the function is called `order`, not `oreder`.